.github/triage/jax_toolbox_triage/local.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import subprocess
+import typing
+
+from .container import Container
+from .utils import run_and_log
+
+
+class LocalContainer(Container):
+    def __init__(self, *, logger: logging.Logger):
+        super().__init__(logger=logger)
+
+    def __enter__(self) -> Container:
+        self._logger.debug("Running local mode inside current container")
+        return self
+
+    def __exit__(self, *exc_info) -> None:
+        pass
+
+    def __repr__(self) -> str:
+        return "Local"
+
+    def exec(
+        self,
+        command: typing.List[str],
+        policy: typing.Literal["once"]
```

Comment:
Python 3.8 doesn't like these type hints - the style used in the other backends should be fine.